src/track/serato/markers2.cpp

```diff
@@ -26,7 +26,9 @@ QString zeroTerminatedUtf8StringtoQString(QDataStream* stream) {
     quint8 byte = '\xFF';
     while (byte != '\x00') {
         *stream >> byte;
-        data.append(byte);
+        if (byte != '\x00') {
```

Comment:
I guess triseratops already does this right?

Reply:
This is actually not a real change. We want to parse a null-terminated string. With Qt 5.x, the `QString::fromUtf8()` stops at the nullbyte, with Qt6 it doesn't which means that `Hello world` becomes `Hello World\u0000`. This just ensures that the QByteArray that we parse the string from doesn't include the trailing nullbyte in the first place. so that the code works with both Qt5 and Qt6.

Since triseratops doesn't use Qt, it's implemented differently anyway.

Reply:
...and the code is so much more concise and readable ;)